vllm/v1/core/sched/scheduler.py

```diff
@@ -923,6 +936,121 @@ def update_from_output(
 
         return engine_core_outputs
 
+    def _update_requests_with_invalid_blocks(
```

Comment:
Could this be moved to `KV Connector Related Methods` section at the bottom of this file